components/iam/BUILD.yaml

```diff
@@ -0,0 +1,45 @@
+packages:
+  - name: app
+    type: go
+    srcs:
+      - "**/*.go"
+      - "go.mod"
+      - "go.sum"
+    deps:
+      - components/common-go:lib
+      - components/gitpod-protocol/go:lib
```

Comment:
We currently don't have any dependency on the `gitpod-protocol`. Do we need to keep this here?